docker-compose/obkm/scripts/deploy.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# ------------------------------------------------------------------------
+# Copyright 2019 WSO2, Inc. (http://wso2.com)
```

Comment:
@Akila94 please update the license header year.

Reply:
Done. Thanks.